src/jobs/helm_promote.yaml

```diff
@@ -1,8 +1,13 @@
 description: "Set version and publish to chart museum"
 
 parameters:
-  version:
-    description: "Name of the helm chart"
+  source_version:
```

Comment:
Why do we need to versions?

Reply:
I want to promote version `0.6.0-RC-1` in `incubator` to `0.6.0` in `stable`

Reply:
Let me see if I understand.
You have a version and you wont republish a new one.
You will kind of import he version from one repository of charts to the other?

If this is it I would let this very clear in the `description` of `helm_promote`. Something in the lines of `Promote version from one chart repository to another (e.g.: incubator -> stable)`

Regarding the `source_version` and `target_version` we can say something along the lines of `Name of the version in the source charts repository`